stackspot-actions/bitbucket/create-repo-bitbucket/script.py

```diff
@@ -0,0 +1,180 @@
+import json
+import requests
+
+
+class Exceptions(Exception):
```

Comment:
Talvez seja melhor renomear essa classe, talvez CreateRepoException ou BitbucketCreateRepoException. Talvez fosse melhor também ao invés de métodos para cada tipo de exeption ter uma subclasse para cada tipo isso facilita o tratamento de uma exception específica ou da generica se for o caso.